src/passwordValidation.js

```diff
@@ -0,0 +1,58 @@
+export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
+
+/**
+ * Checks if a given password is valid or invalid.
+ * If valid it returns true, otherwise false
+ * @param {string} password
+ * @returns {boolean}
+ */
+
+export default function isValidPassword(password = "") {
+  // The following line ensures, that password is always a string, like the number 128 -> string "128"
+  if (typeof password !== "string") password = String(password);
+
+  if (password.length !== 10) {
+    return false;
+  }
+
+  // Check if the password contains at least one number.
+  if (!/\d/.test(password)) {
+    return false;
+  }
+
+  // Check if the password contains at least one letter.
+  if (!/[a-zA-Z]/.test(password)) {
```

Comment:
same here, This can be applied to all the 'if' conditions that return only one line of code.